### ISSUE TITLE
[Feature request] Smooth scrolling through folds

### ISSUE THREAD

**Issue by goingtosleep** [declancm/cinnamon.nvim#8]
First of all, thank you for a very cool plugin! I tried other similar plugins, but none of them can map j and k effectively, which I use a lot (I map `J = 10j` and `K = 10k` for vertical movement instead of `<C-d` and `<C-u`).

So far, so good. Just one thing I feel a bit "missing" is the ability to _scroll through folds_ (and still smooth!). Currently, it feels slow and sticky when going through folds. I would really appreciate it if this issue could be solved.

Thanks again, and have a good day 😄

**Comment by declancm** [comment]
Thank you for liking it! 😁😁

I'll try and get that fixed ASAP but my thesis is due in a week so I will be busy. I'll let you know  as soon as I fix this issue :)

**Comment by declancm** [comment]
Hi @goingtosleep !! Sorry it's taken a while a while...

I've tried changing the way the plugin handles scrolling over folds with the last two commits: 3f6a5f174ea0cbf32f2a6449c604844bfad44a43 555d7b6b18190d66f5c367a660de8911a3bdaa5f . Could you please let me know if this is any better?

Thanks!

**Comment by goingtosleep** [comment]
No worries @declancm, thanks a lot for the update. I used [vim-one](https://github.com/rakr/vim-one) when making this issue, but then I changed to [catppuccin](https://github.com/catppuccin/nvim), and the problem is gone 😅. 

After updating to the latest version, `vim-one` still has non-smooth scrolling over folds, but I guess that's more like something is wrong with my setup. Now I'm using `cinnamon` with `catppuccin` theme and the scrolling is smooth, all good, so I think the issue can be closed.

Thanks again for the cool plugin and have a nice weekend ☺️
